src/com/aca/ArmineArzumanyan/homeWork_1/items/gadget/PenVideoRecorder.java

```diff
@@ -0,0 +1,11 @@
+package com.aca.ArmineArzumanyan.homeWork_1.items.gadget;
+
+public class PenVideoRecorder extends Gadget {
+
+
+    public PenVideoRecorder(String name, int code, int count, int price, int recDuration, GadgetType gadgetType, VideoQuality videoQuality) {
```

Comment:
Actually I didn't like your way to describe the functionality of your items via enum parameters because it adds requirement of validation. For instance, here I can pass GadgetType.AUDIO with VideoQuality.HD which is wrong combination